exchanges/kraken/kraken_test.go

```diff
@@ -1204,61 +1210,218 @@ func TestWithdrawCancel(t *testing.T) {
 
 // ---------------------------- Websocket tests -----------------------------------------
 
-func setupWsTests(t *testing.T) {
-	t.Helper()
+// setupWsAuthTest will connect both websockets
+// and should be called directly from a auth ws test
+func setupWsAuthTest(tb testing.TB) {
+	tb.Helper()
+	setupWsTest(tb)
+	setupAuthWs(tb)
+}
+
+// setupWsTest will just connect the non-authenticated websocket
+// and should be called directly from a non-auth ws test
+func setupWsTest(tb testing.TB) {
+	tb.Helper()
+
+	if !k.Websocket.IsEnabled() {
+		tb.Skip("Websocket not enabled")
```

Comment:
I know a different skip message has been used before.
Unless I'm missing anything, it doesn't hurt to have a more readable message here, but I didn't want to change the original.